extensions/svgl/package.json

```diff
@@ -4,7 +4,10 @@
   "title": "Svgl",
   "description": "Search SVG logos via svgl",
   "icon": "command-icon.png",
-  "author": "davidho0403",
+  "author": "1weiho",
```

Comment:
Why did you change author?

Reply:
The author has already named [1weiho](https://www.raycast.com/1weiho) in Raycast but didn't update it in the extension, causing a pull request action check error. Therefore, I changed the author's name from **davidho0403** to **1weiho**.

Reply:
![CleanShot 2024-03-19 at 20 43 13@2x](https://github.com/raycast/extensions/assets/92412722/87bd382c-7417-424c-b0b4-e7ae79234072)

Reply:
Makes sense, thanks for the clarification 🙂